### ISSUE TITLE
Re-evaluate the design and descriptions of tasks for a better understanding

### ISSUE THREAD

**Issue by AbdBarho** [LAION-AI/Open-Assistant#159]
From comment https://github.com/LAION-AI/Open-Assistant/pull/127#pullrequestreview-1232736041

Consider adjusting the tasks' description and possibly design

For example, the `rank_user_replies` task currently shows different replies, but it does not show what is being replied to? critical context is missing.

Go through all tasks and evaluate whether a change is required and what:
* Assistant reply & User reply
* Rank Assistant replies & Rank User replies
* Rank initial prompts
* Summarize story
* Rank summary

**Comment by yk** [comment]
yes you're right, there should be the entire thread up to that point so the rater knows the context of the replies they're asked to rank!

**Comment by AbdBarho** [comment]
## Assistant reply & User reply


![210204023-25594e51-8c55-4af9-be6a-860e75a0b16f](https://user-images.githubusercontent.com/24505302/210204471-a6c1fdbc-64fe-4e71-916d-efd5ef50fa6b.jpg)


Currently the "speaker" is color coded, maybe it makes sense to add some text to show the speaker? Mockup (design could be better of course):

![Screenshot 2023-01-02 084310](https://user-images.githubusercontent.com/24505302/210204969-8708d9da-eb05-4264-b7a7-c8611aa8c329.jpg)

Maybe using [tags](https://chakra-ui.com/docs/components/tag#with-custom-element) and icons?

**Comment by yk** [comment]
only color-coding is not really suitable, because it needs a legend and potentially people with visual impairments could have trouble discriminating.

"User" is called "Prompter" to avoid confusion internally, but towards the actual user, idk what makes more sense.

I like the approach with avatars the most, maybe even avatars on different sides like many chat-apps

**Comment by othrayte** [comment]
I think avatars is the clearest and most representative of what the conversations actually look like, thus making it the clearest for a new user to understand.

For bonus points we could AI generate some avatars to represent the prompter (kindof a joke, but maybe a lower effort set of N pregenerated avatars wouldn't actually be too much work :smile: ).

**Comment by Jakolo121** [comment]
We can use the "profile Picture" for the prompter. The prompter is already familiar with the Icon. 
![image](https://user-images.githubusercontent.com/34916464/210546642-1e645d8a-721c-4297-9642-7a5702303966.png)

**Comment by fozziethebeat** [comment]
I like where this is going

**Comment by fozziethebeat** [comment]
How are we on this issue? I feel like we haven't discussed this problem in a while so it's probably solved.  @AbdBarho can you evaluate and close if appropriate?

**Comment by fozziethebeat** [comment]
I'm closing this as done, we've overhauled the tasks a lot.
